rsts/concepts/versioning.rst

```diff
@@ -70,29 +68,28 @@ The same ``cache=True`` will handle this complicated situation as well.
 Why Is Versioning Hard?
 -----------------------
 
-Git has become the defacto-standard in version control for code. Git makes it extremely easy to work on branches, merge them, and revert unwanted changes.
-But achieving this for a live (running) algorithm usually needs the entire infrastructure to be associated and potentially re-created for every execution.
+Git has become the defacto-standard in version control for code, making it easy to work on branches, merge them, and revert unwanted changes.
+But achieving this for a live (running) algorithm usually requires the entire infrastructure to be associated and potentially re-created for every execution.
 
-How Is Versioning Tied to Reproducibility?
-------------------------------------------
+How Is Versioning Associated to Reproducibility?
+------------------------------------------------
 
-Reproducibility is possible without explicit versioning within the workflow system.
-To reproduce a past experiment, users need to identify the source code, and resurrect any dependencies that this code may have used (for example, TensorFlow 1.x instead of TensorFlow 2.x, or specific Python libraries).
-It is also necessary to instantiate any infrastructure that the previous version may have used and, if not already recorded, ensure that the previously used dataset (say) can be reconstructed.
-From the first principles, if reproducibility is considered to be one of the most important concerns, then one would capture all these variables and provide them in an easy-to-use method.
+Workflows can be reproduced without explicit versioning within the system.
+To reproduce a past experiment, users need to identify the source code, and resurrect any dependencies that the code may have used (For example: TensorFlow 1.x instead of TensorFlow 2.x, or specific Python libraries).
```

Comment:
```suggestion
To reproduce a past experiment, users need to identify the source code and resurrect any dependencies that the code may have used (for example, TensorFlow 1.x instead of TensorFlow 2.x, or specific Python libraries).
```